db/deploy/mutations/create_rfi.sql

```diff
@@ -0,0 +1,30 @@
+-- Deploy ccbc:mutations/create_rfi to pg
+
+begin;
+
+create or replace function ccbc_public.create_rfi(row_id int)
+returns ccbc_public.rfi_data
+as $function$
+declare
+  result ccbc_public.rfi_data;
+  new_rfi_id int;
+  new_rfi_number varchar(1000); 
+begin
+  select 1 + count(*) into new_rfi_id from ccbc_public.application_rfi_data where application_id=row_id;
+  select CONCAT(ccbc_number,'-',new_rfi_id) into new_rfi_number from ccbc_public.application where id=row_id;
+
+  insert into ccbc_public.rfi_data (rfi_number, json_data, rfi_data_status_type_id) 
+    values (new_rfi_number, '{}'::jsonb, 'draft')
+   returning id into new_rfi_id;
+
```

Comment:
We had a discussion about how we want the RFIs to be created, specifically how we want the applicant to see an RFI. Currently the RFI table will be holding the json data for an RFI form, but the resultant form isn't being created here. Are we planning on using the form_data to create the resultant schema on the applicant side dynamically then?

Reply:
I followed existing approach - application is created with empty json_data, and then gets updated.

Reply:
I meant moreso for the applicant side, the RFI form will only describe the type of schema that the applicant should see without building the schema itself. When an applicant views the RFI, they'll be needing to update or add uploads and we'll need a schema that reflects the RFI created by an analyst. 

It's likely out of scope of this ticket, but the question as to whether the creation of an RFI should build this schema or whether we should reference existing schemas in the `form` table (or some other third solution) isn't being addressed. I'm mostly concerned we sign up with this approach and then are in a corner with regard to how we display the form to the applicant

Reply:
I updated DB schema at https://miro.com/app/board/uXjVOnpFeNI=/ (Frame 3) and my implementation reflects that. My understanding is that RFI would be using schema created in `form` table. I do not think that RFI schema is dynamic in this sense. 
Please feel free to update DB schema on Miro board if you think that I've missed something